view.php

```diff
@@ -52,6 +52,9 @@
 
 $toform['blockid'] = $blockid;
 $toform['courseid'] = $courseid;
+if($block_acclaim_course = $DB->get_record('block_acclaim_courses', array('courseid' => $courseid))){
```

Comment:
Formatting - space after `if` and before `{` (to match both Moodle's and Acclaim's coding standards)

And it's usually considered bad practice to do an assignment in an `if` statement. So:
```
$block_acclaim_course = $DB->get_record('block_acclaim_courses', array('courseid' => $courseid))
if ($block_acclaim_course) {
```